packages/fast-components-react-base/src/typography/examples.data.ts

```diff
@@ -0,0 +1,87 @@
+import { ICategoryItemProps } from "@microsoft/fast-development-site-react";
+import { ISnapshotTestSuite } from "@microsoft/fast-jest-snapshots-react";
+import { IManagedClasses } from "@microsoft/fast-components-class-name-contracts";
+import Typography, {
+    ITypographyHandledProps,
+    ITypographyManagedClasses,
+    ITypographyUnhandledProps,
+    TypeLevel,
+    TypographyTag
+} from "./typography";
+import * as React from "react";
+
+const testString: string = "Test string";
+const managedClassExamples: any = {
+    typography_1: "typography-1",
```

Comment:
When are we doing - as opposed to _ ?

Reply:
this is an example - the actual contract is underscore.